docs/pages/install.mdx

```diff
@@ -2,44 +2,35 @@ import DownloadsComponent from "/components/DownloadsComponent";
 
 # Install Pelican
 
-This document lists the operating system requirements to install Pelican and explains how a user can download Pelican client or Pelican server components.
+This document lists the operating system requirements for Pelican and explains how a user can download and install Pelican.
+
+## Before Starting
+
+Pelican executables can run as a **client** or a **server**.
+
+For client usage, such as accessing the data, you can download and install a Pelican package or Pelican binary [following the instructions below](#install-pelican-binary).
```

Comment:
Maybe "To use the client for interacting with data, download and install the Pelican package or Pelican Binary"?